### PR TITLE
Finishing that PR needed

### PR DESCRIPTION
**Pull request by jgtres**
This PR updates the copy of my slide

##Changes

This resolves #16

### CODE REVIEW

**Comment by jgtres** [comment]
mmm

File: _slides/13-jgtres.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+I have changed it already!
```
Comment (joel8800) [inline _slides/13-jgtres.md:6]: Reviewed!

**Review by joel8800 — APPROVED** [review]

File: _slides/13-jgtres.md (hunk 1 above)
Comment (Raccoon137) [reply _slides/13-jgtres.md:6]: Approved!

**Review by Raccoon137 — APPROVED** [review]
Approved

**Comment by jgtres** [comment]
Merging this PR

**Pull request merged**